docs/application/vstools-mac/overview.md

```diff
@@ -1,150 +1,131 @@
+
+Visual Studio for Mac Extension for Tizen is an extension, which enables you to develop Tizen .NET applications easily using various Tizen project templates with Visual Studio for Mac.
+
+## Prerequisites
+
+To work with Visual Studio for Mac Extension for Tizen, your computer must have:
+
+- At least 5.6 GB of available disk space
+- macOS Mojave version 10.14 (Community, Professional, and Enterprise)
+- Visual Studio 2019 for Mac version 8.2 and higher (Community, Professional, and Enterprise)
+- Java Development Kit (JDK)
+
+  > [!NOTE]
+  > To use Tizen Baseline SDK, you must install [Oracle Java Development Kit(JDK) 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Java for OS X 2015-001](https://support.apple.com/kb/DL1572) together or [OpenJDK 12 and OpenJFX Installation Guide](../tizen-studio/setup/openjdk.md#install-openjdk-for-macos).
+  > 
+  > Ensure that you download and install the exact version. Additionally, for Oracle JDK, Java for OS X 2015-001 is required to support the legacy Java software on macOS.
+
+## Install Visual Studio for Mac Extension for Tizen
+
+To use the Tizen tools in Visual Studio for Mac, you must install:
+
+1. [Visual Studio for Mac Extension for Tizen](#install-tizen-extension)
+2. [Tizen Baseline SDK](#install-tizen-baseline-sdk)
+
+## Install Tizen Extension
+
+To install Visual Studio for Mac Extension for Tizen:
+
+1. In the Visual Studio Mac IDE menu, select **Visual Studio > Extensions**.
+
+   ![Install extension](media/install-extension1.png)
+
+2. In the **Extension Manager** dialog that appears, click **Gallery**.
+
+	![Extension Manager](media/install-extension2.png)
+   
+3. To get the latest extensions, click **Refresh**.
+
+	![Browse Extension](media/install-extension3.png)
+	
+4. After the extension list is refreshed, expand **IDE extensions**, select **Visual Studio for Mac Extension for Tizen**, and click **Install**.
+
+	![Install Popup](media/install-extension4.png)
+	![Install Popup1](media/install-extension5.png)
+
+5. In the **VisualStudio** dialog that appears, click **Install**.
+
+    ![Install Popup2](media/install-extension6.png) 
+
+   The extension is installed. 
+   
+   > [!NOTE]
+   > To complete the installation, ensure that you restart the IDE.
+   
+   After successful installation, **Tizen** appears in the project wizard.
+   
+   ![Project Wizard](media/install-extension7.png)
+
+## Install Tizen Baseline SDK
+
+Baseline SDK contains Tizen-specific libraries and tools such as profiler, debugger, and so on.
+
+After installing the Visual Studio Mac Extension for Tizen, you must set up the Tizen Baseline SDK. You can either:
+
+- [Install a new Tizen Baseline SDK](#install-new-tizen-baseline-sdk) if you have not already installed the SDK.
+- [Configure an existing Tizen Baseline SDK](#configure-existing-tizen-baseline-sdk) If you want to use the installed SDK.
+
+### Install new Tizen Baseline SDK

```

Comment:
I suggested the authors change from `Tizen Studio` to `Tizen baseline SDK`.
I expected to use one of `Tizen baseline SDK` and `Tizen Baseline SDK`. 

I don't have any complain because `Tizen Baseline SDK` is used consistently when I see recent updates.

Reply:
Thank you. Done.